javascript/javascript2/week3/game.js

```diff
@@ -0,0 +1,103 @@
+let gameStart = false;
+let sCount = 0;
+let lCount = 0;
+let counter = 0;
+let isKeyPress = false;
+confetti = new ConfettiGenerator({ target: 's' });
+const h1 = document.getElementById('screen-msg');
+const duration = document.getElementById('duration');
+const button1 = document.getElementById('start');
+const winner = (sCount, lCount) => {
+    if (sCount > lCount) {
+        document.getElementById('msg-s').innerHTML = `Winner`;
+        confetti = new ConfettiGenerator({ target: 's' });
+        confetti.render();
+    }
+    else if (lCount > sCount) {
+        document.getElementById('msg-l').innerHTML = `Winner`;
+        confetti = new ConfettiGenerator({ target: 'l' });
+        confetti.render();
+    }
+    else {
+        document.getElementById('msg-s').innerHTML = `draw`;
+        document.getElementById('msg-l').innerHTML = `draw`;
+
+    }
+}
+
+
+button1.addEventListener('click', countDown => {
+    confetti.clear();
+    let duration1 = parseInt(duration.value);
+    counter = duration1 - 1;
+    if (isNaN(counter)) {
+        document.getElementById('error').innerHTML = 'Please Enter Valid Duration';
+        h1.innerHTML = 'Game Yet To Start';
+        document.getElementById('counter').innerHTML = '';
+        sCount = 0;
+        lCount = 0;
+        document.getElementById('msg-s').innerHTML = '';
+        document.getElementById('msg-l').innerHTML = '';
+        document.getElementById('s-id').innerHTML = sCount;
+        document.getElementById('l-id').innerHTML = lCount;
+
+    }
+    else {
+        sCount = 0;
+        lCount = 0;
+        gameStart = true;
+        document.getElementById('error').innerHTML = '';
```

Comment:
In general, you could possibly organize the code differently, so all the element selectors could be on top, and then only change the required attributes when needed, such as the innerHTML. But regardless of whether you decide to do this change or not, I can definitely understand your code while reading it.

Reply:
I understand 👍🏻